app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,10 +2,40 @@
   <section id="comments" data-follow-button-container="true" data-updated-at="<%= Time.current %>" class="text-padding mb-4 border-t-1 border-0 border-solid border-base-10">
     <% if @article.show_comments %>
       <header class="relative flex justify-between items-center mb-6">
-        <h2 class="crayons-subtitle-1">
-          <%= t("views.articles.comments.subtitle_html",
-                num: tag.span(t("views.articles.comments.num", num: @article.comments_count), class: "js-comments-count", data: { comments_count: @article.comments_count })) %>
-        </h2>
+        <div class="crayons-header__menu" id="comments-sort-dropdown-container">
+          <div class="crayons-subtitle-1" style="display: flex">
+            <h4>
+              <%= t("views.articles.comments.subtitle.#{@comments_order}_html",
+                    num: tag.span(t("views.articles.comments.num", num: @article.comments_count), class: "js-comments-count", data: { comments_count: @article.comments_count })) %>
+            </h4>
+            <button class="crayons-btn crayons-btn--ghost crayons-btn--icon-left" id="toggle-comments-sort-dropdown" aria-controls="comments-sort-dropdown-container" aria-expanded="false" aria-label="Sort comments" aria-haspopup="true">
+              <%= inline_svg_tag("expand.svg", aria_hidden: true, title: t("views.moderations.actions.admin.icon")) %>
+            </button>
+          </div>
+          <nav class="comments-sort-dropdown crayons-dropdown crayons-header__menu__dropdown inline-block" aria-labelledby="sort-comments-heading">
+              <h4 class="comments-sort-title">Sort discussion: </h4>
+              <ul class="comments-sort-dropdown__list">
+                <%= render "comments/sort_option",
+                           sort_order: "top",
+                           sort_title: "Top",
+                           sort_description: "Most upvoted and relevant comments will be first",
+                           show_selected: @comments_order == "top",
+                           html_classes: "first-nav-link" %>
+                <%= render "comments/sort_option",
+                           sort_order: "latest",
+                           sort_title: "Latest",
+                           sort_description: "Most recent comments will be first",
+                           show_selected: @comments_order == "latest",
+                           html_classes: "second-last-nav-link" %>
+                <%= render "comments/sort_option",
+                           sort_order: "oldest",
+                           sort_title: "Oldest",
+                           sort_description: "The oldest comments will be first",
+                           show_selected: @comments_order == "oldest",
+                           html_classes: "last-nav-link" %>
+              </ul>
+          </nav>
+        </div>
```

Comment:
```suggestion
<div class="flex">
          <h2>
            <%= t("views.articles.comments.subtitle.#{@comments_order}_html",
                num: tag.span(t("views.articles.comments.num", num: @article.comments_count), class: "js-comments-count", data: { comments_count: @article.comments_count })) %>
          </h2>
          <button id="toggle-comments-sort-dropdown" class="crayons-btn crayons-btn--ghost crayons-btn--icon-left" id="toggle-comments-sort-dropdown" aria-controls="comments-sort-dropdown-container" aria-expanded="false" aria-label="Sort comments" aria-haspopup="true">
            <%= inline_svg_tag("expand.svg", aria_hidden: true, title: t("views.moderations.actions.admin.icon")) %>
          </button>
        </div>
        <nav id="comments-sort-dropdown-container" class="crayons-dropdown p-4" aria-labelledby="sort-comments-heading">
          <h4 class="comments-sort-title">Sort discussion: </h4>
          <ul class="comments-sort-dropdown__list">
            <%= render "comments/sort_option",
                        sort_order: "top",
                        sort_title: "Top",
                        sort_description: "Most upvoted and relevant comments will be first",
                        show_selected: @comments_order == "top",
                        html_classes: "first-nav-link" %>
            <%= render "comments/sort_option",
                        sort_order: "latest",
                        sort_title: "Latest",
                        sort_description: "Most recent comments will be first",
                        show_selected: @comments_order == "latest",
                        html_classes: "second-last-nav-link" %>
            <%= render "comments/sort_option",
                        sort_order: "oldest",
                        sort_title: "Oldest",
                        sort_description: "The oldest comments will be first",
                        show_selected: @comments_order == "oldest",
                        html_classes: "last-nav-link" %>
          </ul>
        </nav>
```

Here I have again refactored to remove any conflation with the member top navigation menu, as mentioned in the `commentDropdowns` file. Removing the conflation allows the dropdown to be opened/closed, relying on the `crayons-dropdown` class and without any strange stacking issues (it is expected that the open dropdown would appear over the sticky header when scrolled):

https://user-images.githubusercontent.com/20773163/160593995-3315e4bb-82b6-44e7-9ce9-5b630ecdef6b.mp4